contrib/test.sh

```diff
@@ -22,6 +22,15 @@ if cargo --version | grep nightly; then
     NIGHTLY=true
 fi
 
+# We should not have any duplicate dependencies. This catches mistakes made upgrading dependencies
+# in one crate and not in another (e.g. upgrade bitcoin_hashes in bitcoin but not in secp).
+duplicate_dependencies=$(cargo tree  --target=all --all-features --duplicates | wc -l)
+if [ $duplicate_dependencies -ne 0 ]; then
```

Comment:
Bash paranoia mindset says you should put double quotes around variables (unless intentional). In this case if `wc` screwed up and printed a space. (Unlikely but a good habit.)

Reply:
Sure thing, can do. Cheers.